src/java/fr/paris/lutece/portal/business/user/AdminUserHome.java

```diff
@@ -113,6 +115,7 @@ public static Collection<AdminUser> findUserList( )
     public static void create( AdminUser user )
     {
         _dao.insert( user );
+        AppLogService.debug( "User created : User id = {}, User access code = {}", ( ) -> user.getUserId( ), ( ) -> user.getAccessCode( ) );
     }
```

Comment:
Avoid logging sensitive information.

Reply:
I will remove the user access code (I decided to add it because this data is displayed in accessLogger.log but no problem I will remove it from this code). Do you want that I remove the user id also ?